CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 ### Improvements
 + Replaced dependency `find-up` with `findup-sync` in `algob`.
 + Added `algopy` in `@algo-builder/algob/sample-project`, which enables users to pass template parameters to PyTEAL contracts. Updated docs.
++ Store checkpoints in nested form for SSC, added tests.
```

Comment:
do we need to store checkpoints in subdirectory? How about applying a more complex names? eg: `subdir_name-filename-timestamp.yml`

Reply:
No, this point is not related to this PR. This point was in the released version (updating checkpoint structure for *updateSSC*) but it's not released yet, so moved it here.